packages/context-editor/src/stories/AtomRenderer.tsx

```diff
@@ -1,13 +1,16 @@
-import type { Node } from "prosemirror-model";
+import type { NodeViewComponentProps } from "@handlewithcare/react-prosemirror";
 
-import React, { Suspense, useEffect, useState } from "react";
-import { useNodeViewContext } from "@prosemirror-adapter/react";
+import React, { forwardRef, useEffect, useState } from "react";
+import { useIsNodeSelected } from "@handlewithcare/react-prosemirror";
 import { CsvToHtmlTable } from "react-csv-to-table";
 
-export default function ContextAtom({ nodeProp }: { nodeProp: Node }) {
-	const { contentRef, node, selected } = useNodeViewContext();
+export const AtomRenderer = forwardRef<
+	HTMLDivElement,
+	NodeViewComponentProps & { selected?: boolean }
+>(function AtomRenderer({ nodeProps, selected, ...props }, ref) {
+	const { node } = nodeProps;
```

Comment:
See https://github.com/pubpub/platform/pull/1159/files#r2047460927 for explanation of this change.